server/openslides/agenda/models.py

```diff
@@ -383,7 +390,7 @@ class ListOfSpeakers(RESTModelMixin, models.Model):
     Field for generic relation to a related object. General field to the related object.
     """
 
-    closed = models.BooleanField(default=False)
+    closed = models.BooleanField(default=list_of_speakers_initially_closed)
```

Comment:
This changes required a new migration file (python manage.py makemigrations). Or maybe we found an alternative to set this list-of-speakers toggle (e.g. on client side)?

Reply:
AN alternative would be to add it here: https://github.com/OpenSlides/OpenSlides/blob/master/server/openslides/agenda/signals.py#L88 This is the single place lists of speakers are created and do not add migrations.

Reply:
After looking into these options, setting the field when the `ListOfSpeakers` object is created still seemed the cleanest solution to me, so I added a migration file.